src/ls/definition/html_element.cr

```diff
@@ -0,0 +1,19 @@
+module Mint
+  module LS
+    class Definition < LSP::RequestMessage
+      def definition(node : Ast::HtmlElement, server : Server, workspace : Workspace, stack : Array(Ast::Node))
+        node.styles.each do |style|
+          next unless cursor_intersects?(style)
+
+          return definition(style, server, workspace, stack)
+        end
+
+        node.attributes.each do |attribute|
```

Comment:
I think this is unnecessary since attributes for HTML elements are not defined anywhere for example `onClick` or `method` for `form`, but maybe I'm missing something :slightly_smiling_face:

Reply:
Yes you are completely correct, only `styles` needs to be checked here.

It's actually doubly redundant as `Ast::HtmlAttribute` is a [tracked node](https://github.com/mint-lang/mint/blob/master/src/parsers/html_attribute.cr#L31), which is why it still works for `Ast::HtmlComponent` without any similar logic. :sweat_smile: